cds/modules/records/serializers/schemas/project.py

```diff
@@ -75,6 +74,7 @@ class ProjectSchema(StrictKeysSchema):
     title_translations = fields.Nested(TitleTranslationSchema, many=True)
     videos = fields.Nested(ProjectVideoSchema, many=True)
 
-    category = fields.Str()
-    type = fields.Str()
+    date = fields.Str()
+    category = fields.Str(required=True)
+    type = fields.Str(required=True)
```

Comment:
IMHO the ``content_type`` is a bit misleading :/ We are using the ``type`` in [other places too](https://github.com/CERNDocumentServer/cds/blob/cdslabs_qa/cds/modules/records/serializers/schemas/project.py#L47) 

WDYT about keeping the ``type`` attribute for consistency and fix the builtin issue as @omelkonian suggested?